docs/apple_app_signing.md

```diff
@@ -1 +1,208 @@
-Apple Signing
\ No newline at end of file
+# Apple Signing
+
+To learn about code signing, please review Apple's [code signing](https://help.apple.com/xcode/mac/current/#/devfbe995ebf) documentation. 
+
+This guide details how to work with signing certificates.
+
+## Certificates
+Apple provides two types of certificates:
+
+* Development
+* Distribution
+
+
+Both certificate types need the app's [bundle id](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids). It was created during the project's onboarding phase. Please see your Product or Technical Owner if you do not have the bundle id. 
+
+ 
+### Development
+
+Developer accounts are set to have access to development certificates, identifiers & profiles.  You can access them within Xcode by setting the team to the Province of BC. Please review Apple's [Assign a project to a team](https://help.apple.com/xcode/mac/current/#/dev23aab79b4) documentation for details on how to do this. Turn on the "automatically manage signing" in the Signing & Capabilities -> Debug tab.
+
+Developers may install the app on their own device. This can be [setup through Xcode](https://developer.apple.com/documentation/xcode/distributing-your-app-to-registered-devices#Register-devices-automatically-in-Xcode). Please use [TestFlight](https://testflight.apple.com) if you need to test on many devices.
```

Comment:
```suggestion
Developers may install the app on their own device. This can be [setup through Xcode](https://developer.apple.com/documentation/xcode/distributing-your-app-to-registered-devices#Register-devices-automatically-in-Xcode). Use [TestFlight](https://testflight.apple.com) if you need to test on many devices.
```